packages/e2e-tests/specs/taxonomies.test.js

```diff
@@ -31,7 +31,7 @@ describe( 'Taxonomies', () => {
 		return page.evaluate(
 			() => {
 				return Array.from( document.querySelectorAll(
-					'.editor-post-taxonomies__hierarchical-terms-input:checked'
+					'.editor-post-taxonomies__hierarchical-terms-choice .components-checkbox-control__input:checked'
```

Comment:
The structure of the html is different now, there's an extra wrapper around the `input`, so on line 36 where there's a call to `node.parentElement.querySelector( 'label' )`, there's no label within the parentElement.

Could try using `node.closest` instead of `node.parentElement` to select the parent of the label (https://developer.mozilla.org/en-US/docs/Web/API/Element/closest).